src/h5web/visualizations/containers/NxSpectrumContainer.tsx

```diff
@@ -31,9 +32,12 @@ function NxSpectrumContainer(props: VisContainerProps): ReactElement {
     'x',
   ]);
 
-  const value = useDatasetValue(dataset.id);
+  const values = useDatasetValues([
+    dataset.id,
+    ...axesDatasets.map((d) => d && d.id),
```

Comment:
If the dataset was not found, return an `undefined` id to keep the numbers of elements in the array (https://github.com/silx-kit/h5web/pull/256/files#r509287208). This means `useDatasetValues` and its Promise-based version `getValues` must accept `undefined` arguments (https://github.com/silx-kit/h5web/pull/256/files#r509288853)